src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -265,7 +275,8 @@ private void autoPeriodic() {
     logger.recordOutput("Arm/AutoMode", autoMode.toString());
     switch (autoMode) {
       case RETRACT:
-        if (armIOInputs.extendPosition < SAFE_RETRACT_NON_HOME + EXTEND_TOLERANCE_METERS) {
+        if (autoRetractTimer.hasElapsed(1)
```

Comment:
Does this timer impl reset and start? Timers should be used in either periodic or a command

Reply:
Yes, it has reset() and start() in setTargetPositions.